examples/format_seqFISH.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, input_dir: str) -> None:
         self.input_dir = input_dir
 
     @property
-    def coordinates(self) -> Mapping[Coordinates, Tuple[float, float]]:
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
```

Comment:
If this is the fix, mypy is being a bit overactive here, no?

Reply:
I was certainly surprised myself. I tried a couple of different ways of assigning `X` to `Union[X, Y]` to no avail. I'm certainly no mypy expert though. There were suggestions of making the union an alias used everyone, which isn't really what you're trying to achieve. Then there's [@overload](https://stackoverflow.com/a/49101404/56887) but since it's not used in the current codebase, I didn't try that route.

Reply:
No, this makes sense.  Parameters for subclasses can accept a broader type definition, but not a narrower one.  If mypy is built correctly, return values should behave in the opposite way.

Reply:
Oh I see. That makes a lot of sense, thanks @ttung . I wasn't considering the subclass angle.